### PR TITLE
Feature/list employee rewards in admin panel

### PR DESCRIPTION
**Pull request by K4sku**
Add display of employee's rewards to admins_employee show action

PR includes:
- nemoization in admins_employee controller
- new route for admins_employee_path(employee)
- show action in controller
- show link in admin_employees index view
- show admins_employee_path(employee) view
- system tests
admins_employee_show

### CODE REVIEW

File: app/controllers/admins/employees_controller.rb [hunk 1]
```diff
@@ -4,29 +4,35 @@ def index
...
       redirect_back fallback_location: kudos_path
     end
 
     private
 
+    def employee
+      @employee ||= Employee.find(params[:id])
+    end
+
     def find_employee
```
Comment (ciejj) [inline app/controllers/admins/employees_controller.rb]: do we need this method?

File: spec/system/admins/employee/index_spec.rb [hunk 2]
```diff
@@ -25,5 +25,10 @@
       visit admins_employees_path
       expect(page).to have_link('Edit', count: 2)
     end
+
+    it 'shows show links' do
```
Comment (ciejj) [inline spec/system/admins/employee/index_spec.rb]: maybe:
```suggestion
    it 'displays show links' do
```

File: spec/system/admins/employee/show_spec.rb [hunk 3]
```diff
@@ -0,0 +1,28 @@
+require 'rails_helper'
+
+describe 'admins see employees rewards', type: :system, js: true do
+  before do
+    create(:employee)
+    create(:kudo, reciever: employee)
+    # create(:order, employee: employee, purchase_price: reward.price)
```
Comment (ciejj) [inline spec/system/admins/employee/show_spec.rb]: unnecessary comment

**Review by ciejj — APPROVED** [review]
Well done, not much to add from my side :)

File: spec/system/admins/employee/show_spec.rb (hunk 3 above)
Comment (K4sku) [reply spec/system/admins/employee/show_spec.rb]: removed

File: spec/system/admins/employee/index_spec.rb (hunk 2 above)
Comment (K4sku) [reply spec/system/admins/employee/index_spec.rb]: this seemed odd to me too. Changed

File: app/controllers/admins/employees_controller.rb (hunk 1 above)
Comment (K4sku) [reply app/controllers/admins/employees_controller.rb]: Not anymore, removed. Thanks for CR

**Pull request merged**